src/ewatercycle_wflow/forcing.py

```diff
@@ -28,6 +28,7 @@ class WflowForcing(DefaultForcing):
         Inflow (str) = None: Variable name of inflow data in input file.
     """
 
+    filenames: dict[str, str] = {}
```

Comment:
I chose to leave this empty as wflow (old python version) requires a single netcdf for multiple variables. It had to be included though to follow the parent class.

Reply:
weird that extending needs you to repeat the properties, oh well.

Reply:
Yeah not sure, didn't have to do this with Hype. I'll double check...

Reply:
Oh it runs fine without that line now :man_facepalming: